tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_heartbeat.c

```diff
@@ -98,21 +143,40 @@ static void publish_common(void)
 		.period = PUBLISH_PERIOD_SEC,
 		.net_idx = 0,
 		.ttl = PUBLISH_TTL,
+		.feat = BT_MESH_FEAT_SUPPORTED
```

Comment:
I recon there is a valid reason for initializing this parameter, but does not the heartbeat module overwrite this parameter anyways?